src/extendedstorage.cpp

```diff
@@ -393,9 +307,7 @@ void ExtendedStorage::setModified(const QString &info)
         }
     }
     calendar()->close();
-    clearLoaded();
-    d->mNotebooks.clear();
-    d->mDefaultNotebook.clear();
+    d->close();
```

Comment:
I don't like much the `info` argument but it's part of the public observer API and I'm not keen to delete it yet.  Would need some propagation to nemo-qml-plugin-calendar. Maybe for a later API break commit, all of those of the current PR should not require any propagation of API changes.

What I can do though is to rename these vague names with the name of the observer signal and an `emit` verb. Like that `setModified()` would become `emitStorageModified()`. It would make the purpose of calling such method clear in the implementations.